src/core_plugins/tile_map/public/geohash_layer.js

```diff
@@ -9,15 +9,14 @@ import { GeohashGridMarkers } from './markers/geohash_grid';
 
 export class GeohashLayer extends KibanaMapLayer {
 
-  constructor(featureCollection, options, zoom, kibanaMap) {
+  constructor(featureCollectionAndMeta, options, zoom, kibanaMap) {
```

Comment:
why don't you pass these in as two separate parameters? Why are they squished into a single object?

Reply:
you're right, looks wonky. Will change. similar for your other comment.